cpplint_unittest.py

```diff
@@ -320,8 +342,16 @@ def testFalsePositivesNoError(self):
   # Test get line width.
   def testGetLineWidth(self):
     self.assertEquals(0, cpplint.GetLineWidth(''))
+<<<<<<< b2e46ca571a1cff5431a12a5cac8e4fc856ce852
     self.assertEquals(10, cpplint.GetLineWidth('x' * 10))
     self.assertEquals(16, cpplint.GetLineWidth(u('\u90fd|\u9053|\u5e9c|\u770c|\u652f\u5e81')))
+=======
+    self.assertEquals(10, cpplint.GetLineWidth(unicode('x') * 10))
+    try:
+      self.assertEquals(16, cpplint.GetLineWidth('都|道|府|県|支庁'.decode('utf-8')))
+    except AttributeError:
+      self.assertEquals(16, cpplint.GetLineWidth('都|道|府|県|支庁'))
+>>>>>>> Added support for Python3000
```

Comment:
You _have_ to use the unicode around the `'x'` as that is what the test is testing. `self.assertEquals(10, cpplint.GetLineWidth(u'x' * 10))` is equal to 'ten of the `'x'` character in Unicode is 10 bytes wide'. Changing the line of code to `'x' * 10` results in a different test on Python2 and Python3. Python2 will test that a `'x'` _byte_ multiplied by 10 is a width of 10 and on Python3 `'x'` in Unicode code point is 10 in width.
The unicode notation is nicer, single line.